packages/language-server/test/ui5-model-spec.ts

```diff
@@ -29,5 +41,110 @@ describe("the UI5 language assistant ui5 model", () => {
     expect(Object.keys(ui5Model.enums)).to.include(
       "sap.ca.ui.charts.ChartSelectionMode"
     );
-  }).timeout(5000); // The default timeout is 2000ms and this test takes ~3000ms
+  }
+
+  it("will get UI5 semantic model", async () => {
+    const ui5Model = await getSemanticModel(undefined);
+    assertSemanticModel(ui5Model);
+  }).timeout(GET_MODEL_TIMEOUT);
+
+  it("doesn't fail if a file cannot be fetched", async () => {
+    const ui5Model = await getSemanticModelWithFetcher(async (url: string) => {
```

Comment:
why won't this fail? does it not fail because the previous test populated a cache?

Reply:
The tests don't rely on the order - the previous test doesn't populate the cache because we don't send a cache folder. It doesn't fail because the code is fault-tolerant. It will write the error to the output and do its best with what it could get instead of not working at all.

Reply:
I'll make it more explicit that we don't populate the cache in these 2 tests